README.md

```diff
@@ -1 +1,45 @@
-# gitflow
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# gitflow
```

Comment:
- [ ] [OPTIONAL] Add an appropriate project title. This will give an overall of the project.